AdminUi/test/AdminUi.Tests.Integration/Models/HttpResponse.cs

```diff
@@ -10,3 +10,11 @@ public class HttpResponse<T>
     public string? ContentType { get; set; }
     public string? RawContent { get; set; }
 }
+
+public class HttpResponse
```

Comment:
`HttpResponse<T>` could inherit from `HttpResponse`

Reply:
But then we would have an error property on both the HttpResponse and the Content of HttpResponse<T>, hence why I didn't use inheritance. Thoughts?